MakeHist/HTLT.C

```diff
@@ -98,4 +128,100 @@ void HTLT(){
 
     leg_HTLT->Draw();
 
+    c1->SaveAs("./hists/"+prefix+"_HTLT.png");
+
+
+    //make bkg for siginificance plot
+    HTLT_DB->Add(HTLT_ST);
+    HTLT_DB->Add(HTLT_TT);
+    HTLT_DB->Add(HTLT_DY);
+
+    TH1F * HTLT_DB_copy = (TH1F*) HTLT_DB->Clone("DB_copy");
+
+    TH1F * HTLT_200_copy = (TH1F*) HTLT_200->Clone("200_copy");
+
+    //std::cout << "ComputeIntegral " << HTLT_DB->ComputeIntegral() << "\n";
+    float previousBin;
+    float currentBin;
+    int nBins = HTLT_DB->GetNbinsX();
+
+    //std::cout << "right cut \n";
+    for(int i=-2;i<nBins+3;i++){
```

Comment:
Please explain the range of the loop and how the `GetBinContent` behaves when you call it for bin number "-2"?